models/src/main/thrift/content/v1.thrift

```diff
@@ -1206,6 +1206,8 @@ struct Package {
 
     /* The package name */
     3: required string packageName
+
+    4: optional CapiDateTime lastModified
```

Comment:
@tomrf1 I was under the same impression as Mariot. So if @cb372 thinks it can be marked as required without causing compatibility issues, then it should be `required`.